engine/Shopware/Models/Document/Document.php

```diff
@@ -55,6 +55,16 @@ class Document extends ModelEntity
      */
     private $name = '';
 
+    /**
```

Comment:
"freely choseable [sic]" is probably not a good way to put this, I'd leave this part out.